tests/TestCase.php

```diff
@@ -23,6 +23,8 @@ public function setUp()
 
         $this->setUpRoutes($this->app);
 
+        $this->registerFallbackRoute();
```

Comment:
Do we really need to register this callback manually in the tests? I'm thinking that the `DemoModeServiceProvider` would already to care of this, no?

Reply:
It appears that we do. You wouldn't think so. (The suite fails without this - even though I have confirmed that the route is indeed registered in the service provider via the browser).

Let me know if you have any ideas regarding this.